release_announcement

```diff
@@ -15,7 +15,7 @@ fi
 echo >> "$ANNOUNCEMENT"
 
 if [[ $PROJECT == foreman ]] ; then
-	# TODO: inform the user input is expected?
+	echo Enter the announcement text, end by CTRL+D: >&2
```

Comment:
A very nitpicky thing: technically someone could pipe data in and `cat` will consume that. However, given the context that this script is used in I think it's fine to ignore that edge case now. At least the message it put on stderr so you can still copy the output with a pipe and that's more important.